### ISSUE TITLE
links aren't discoverable to screen reader users in markdown documents

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#211657]
They're only discoverable via visual distinction and the action that can be taken (IE opening them) is only indicated in the tooltip AFAICT.

https://github.com/microsoft/vscode/assets/29464607/09d28b81-c2cc-4477-b1fc-7b1de1baae74

**Comment by meganrogge** [comment]
Furthermore, one cannot action these via keyboard - we don't have a default keybinding assigned for `editor.action.openLink`.
